scripts/genMaterial.js

```
@@ -1 +1,801 @@
+  const slotNames = new Set(); // 使用 Set 去重
+
+  let match;
+  while ((match = regex.exec(funcString)) !== null) {
+    slotNames.add(match[1]); // 添加 x 到集合中
+  }
+
+  return Array.from(slotNames); // 转换为数组
+}
+
+function getLabelName(key, labelName) {
+  if (labelName !== EMPTY_VALUE) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)